hack/make-rules/test.sh

```diff
@@ -263,6 +265,7 @@ runTests() {
           ${testargs[@]:+${testargs[@]}} \
         | tee ${junit_filename_prefix:+\"${junit_filename_prefix}-\$_pkg_out.stdout\"} \
         | grep \"${go_test_grep_pattern}\"" \
+          {} \
```

Comment:
I would defer to @ixdy for full bash style, but probably 2 spaces under bash, since it's a continuation of that command